core-initrd/latest/bin/ubuntu-core-initramfs

```diff
@@ -758,8 +782,9 @@ def create_initrd(parser, args):
             )
         check_call(["depmod", "-a", "-b", main, args.kernelver])
 
-        if not verify_missing_dlopen(main, os.path.join("/usr/lib", deb_arch)):
-            sys.exit(1)
+        if ubuntu_release[0] >= 25 or (ubuntu_release[0] >= 24 and ubuntu_release[1] >= 10):
```

Comment:
should this check be a helper given we repeat it twice?

Reply:
I've created an ancillary function for this. Note though that checks might be in the future for different Ubuntu releases (we might need to do some things only from 25.04 onwards, etc.)